### DISCUSSION TITLE
New artisan command

### DISCUSSION THREAD

**Discussion by kingdomac** [laravel/framework#46980] [Ideas]
Laravel idea:
Isn't easier if we had a universal artisan command 'make' wich will generate classes based on last prefix:

("classNameController", "...Model", "...Request", "...Resource", "...Factory", "...Seeder", "...Policy", "...Mail", "...Notification", "...Event", "...Job", "...Listener" ...) It helps to simplify the command, and  also create some name convention to follow.

php artisan make UserController
php artisan make UserFactory
php artisan make UserSeeder
....

**Comment by TheBlckbird** [comment]
It could simplify the command but you shouldn’t name models `{Name}Model`. The same applies for a few other of these classes.

**Reply by kingdomac** [reply]
It is important to specify model either controller or  ... Just to  seperate the logic  from arbitrary classes creation

**Comment by shaedrich** [comment]
I don't think, it should be either or (keeping in mind what @TheBlckbird said for example) but it wouldn't hurt introducing this as a shortcut when possible, I suppose.
